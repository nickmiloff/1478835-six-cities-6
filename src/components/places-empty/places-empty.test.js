import React from 'react';
import {render} from '@testing-library/react';

import PlacesEmpty from './places-empty';

describe(`PlacesEmpty component test`, () => {
  it(`PlacesEmpty component' should render correctly`, () => {
    const LOCATION = `Paris`;

    const {container} = render(<PlacesEmpty location={LOCATION} />);

    expect(container).toMatchSnapshot();
  });
});
