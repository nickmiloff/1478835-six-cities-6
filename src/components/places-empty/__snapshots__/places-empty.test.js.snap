// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesEmpty component test PlacesEmpty component' should render correctly 1`] = `
<div>
  <section
    class="cities__no-places"
  >
    <div
      class="cities__status-wrapper tabs__content"
    >
      <b
        class="cities__status"
      >
        No places to stay available
      </b>
      <p
        class="cities__status-description"
      >
        We could not find any property available at the moment in 
        Paris
      </p>
    </div>
  </section>
</div>
`;
